ingestion/src/metadata/ingestion/ometa/mixins/table_mixin.py

```diff
@@ -215,3 +218,58 @@ def create_or_update_table_profiler_config(
             )
 
         return None
+
+    def get_profile_data(
+        self,
+        fqn: str,
+        start_ts: int,
+        end_ts: int,
+        limit=100,
+        after=None,
+        profile_type: Union[Type[TableProfile], Type[ColumnProfile]] = TableProfile,
+    ):
+        """Get profile data
+
+        Args:
+            fqn (str): fullyQualifiedName
+            start_ts (int): start timestamp
+            end_ts (int): end timestamp
+            limit (int, optional): limit of record to return. Defaults to 100.
+            after (_type_, optional): use for API pagination. Defaults to None.
+            profile_type (Union[Type[TableProfile], Type[ColumnProfile]], optional):
+                Profile type to retrieve. Defaults to TableProfile.
+
+        Raises:
+            ValueError: _description_
+            TypeError: _description_
+
+        Returns:
+            _type_: Profile object with data, after and total attributs
```

Comment:
If we add this info as a docstring could we also add the type hint in the definition?

Reply:
ok just saw we use a `namedtuple`. Any reason we're not defining a BaseModel that we can use and share better?

Reply:
They are slightly different for each DB.

Reply:
then could we have explicit definitions for each db and here pass the parent type? This way it might be easier to understand what we need in each scenario. We won't support all DBs out of the box, so we could just keep expanding the defs as needed

Reply:
Sorry i thought we were talking about something else. There is actually already something defined [here](https://github.com/open-metadata/OpenMetadata/blob/25449001ca978fd14bd7178a194cb40402bb4f6a/ingestion/src/metadata/ingestion/ometa/ometa_api.py#L131-L143) though we can't use it in the Mixins as it throws a circular import. We could move that base class but i am not sure where it actually fits. 

The other solution is to move this method out of the mixin and insidie `ometa_api`

Reply:
in any case, I believe this is important enough to have a closed definition. It can save some headaches down the line

Reply:
we can close this PR if you want and handle this in another iteration, don't want to hold this off just because of this though